common/src/main/java/org/dash/wallet/common/util/GenericUtils.java

```diff
@@ -200,6 +200,21 @@ public static String fiatToString(Fiat fiat) {
         }
     }
 
+    public static boolean isCurrencyFirst(Fiat fiat) {
+        NumberFormat numberFormat = NumberFormat.getCurrencyInstance(getDeviceLocale());
+        Currency currency = Currency.getInstance(fiat.currencyCode);
+        numberFormat.setCurrency(currency);
+        String currencySymbol = currency.getSymbol(getDeviceLocale());
+       return numberFormat.format(1.0).startsWith(currencySymbol);
+    }
+
+    public static String getLocalCurrencySymbol(String currencyCode) {
+        NumberFormat numberFormat = NumberFormat.getCurrencyInstance(getDeviceLocale());
+        Currency currency = Currency.getInstance(currencyCode);
+        numberFormat.setCurrency(currency);
+        return currency.getSymbol(getDeviceLocale());
+    }
```

Comment:
This will be one merge conflict with Claude's PR:

It is possible for some phones to above old Locale information for certain currencies that have changed (eg Belarus).
```
    public static String getLocaleCurrencyCode(){
        Currency currency = Currency.getInstance(getDeviceLocale());
        String newCurrencyCode = currency.getCurrencyCode();
        if (CurrencyInfo.hasObsoleteCurrency(newCurrencyCode)) {
            newCurrencyCode = CurrencyInfo.getUpdatedCurrency(newCurrencyCode);
        }
        newCurrencyCode = CurrencyInfo.getOtherName(newCurrencyCode);
        return newCurrencyCode;
    }
```

Reply:
I tried @ClaudeHangui  method we need the local currency Symbol and the one claude create return code

Reply:
@hadia @ClaudeHangui I suppose it is fine.